frontend/src/modules/contributor/components/edit/work-history/contributor-work-history-edit.vue

```diff
@@ -242,6 +254,27 @@ const isModalOpen = computed<boolean>({
   },
 });
 
+const hasSameOrganization = computed(() => props.contributor.organizations.some((o: Organization) => o.id === form.organization?.id)
+    && (!isEdit.value || form.organization?.id !== props.organization?.id));
+
+const hasSameOrgDetails = computed(() => props.contributor.organizations
+  .some((o: Organization) => {
+    if (o.id !== form.organization?.id) {
+      return false;
+    }
+    if (form.title === o.memberOrganizations.title) {
+      return true;
+    }
+    console.log(form.dateStart, o.memberOrganizations.dateStart);
+    const start = form.dateStart && o.memberOrganizations.dateStart
+      ? moment(form.dateStart).startOf('month').isSame(moment(o.memberOrganizations.dateStart), 'day')
+      : form.dateStart === o.memberOrganizations.dateStart;
+    const end = !form.currentlyWorking && form.dateEnd && o.memberOrganizations.dateEnd
+      ? moment(form.dateEnd).startOf('month').isSame(moment(o.memberOrganizations.dateEnd), 'day')
+      : form.dateEnd === o.memberOrganizations.dateEnd;
+    return start && end;
+  }));
```

Comment:
I don't think this is ok considering my last comment https://github.com/CrowdDotDev/crowd.dev/pull/2546/files#r1692949392 I think this method should be updated to something like (added comments so that it's also clearer):

```
const hasInvalidFormValues = computed(() => props.contributor.organizations
  .some((o: Organization) => {
    // Organization doesn't yet exist as a work experience
    if (o.id !== form.organization?.id) {
      return false;
    }

    // Organization already exists as a work experience
    // AND job title is from the existing work experience
    if (form.title !== o.memberOrganizations.title) {
      return false;
    }

    // Organization already exists as a work experience
    // AND both job title and period are empty
    if (!form.title && !form.dateStart && !form.dateEnd) {
      return true;
    }

    // Organization already exists as a work experience
    // AND period is the same as the existing work experience
    const start = form.dateStart && o.memberOrganizations.dateStart
      ? moment(form.dateStart).startOf('month').isSame(moment(o.memberOrganizations.dateStart), 'day')
      : form.dateStart === o.memberOrganizations.dateStart;
    const end = !form.currentlyWorking && form.dateEnd && o.memberOrganizations.dateEnd
      ? moment(form.dateEnd).startOf('month').isSame(moment(o.memberOrganizations.dateEnd), 'day')
      : form.dateEnd === o.memberOrganizations.dateEnd;

    return start && end;
  }));
```